src/components/TokenBalance.tsx

```diff
@@ -223,7 +222,11 @@ export function AssetsTokenBalance({
             </TouchableOpacity>
           )}
         </View>
-        <TokenBalance style={styles.totalBalance} singleTokenViewEnabled={false} />
+        <TokenBalance
+          style={styles.totalBalance}
+          singleTokenViewEnabled={false}
+          showBalanceToggle={true}
```

Comment:
```suggestion
          showBalanceToggle={isWalletTab}
```
otherwise this will show up on the old Assets page